script/Arcueid/Day 3/03_00_ARC03_6.txt

```diff
@@ -322,7 +329,9 @@ It was all a bad dream.
 [sha:317fca2529581594423d05ed8b4e96d0b657ec4f]{
 -- Page 293, Offset 4806.
 -- 意味もなく理由もなく、幼い頃に先生と交わした約束を破って、俺があんなコトをするわけがなかったんだ。
-It's not like I'd do something like that without meaning or reason and break a promise I made with Sensei when I was little.
+// 意味 and 理由 repetition spread out a bit
+There's no way I'd senselessly go and break the promise I made to Sensei when I was little.
+ Not without having any reason like that.
```

Comment:
Not without having even a single reason.

I don't like the 'any reason' phrasing